src/@next/Menu/Menu.tsx

```diff
@@ -0,0 +1,56 @@
+import React, { useCallback } from 'react';
+import { Typography } from '../Typography';
+import { Neutral } from '../utilities/colors';
+import { ListContainer, StyledMenu, TitleContainer } from './MenuStyle';
+
+interface Option {
+  active?: boolean;
+  disabled?: boolean;
+  id?: string;
+  label: React.ReactNode;
+  value: string;
+}
+export interface MenuProps {
+  id?: string;
+  onChange?: (selected: string[]) => void;
+  options?: Option[];
+  selected?: string[];
```

Comment:
Probably need to make it clear this will relate to `Option.value`, maybe we should rename this as `selectedValue` and add a description

Reply:
agree! renamed and added definition